{"loss": "<keras.src.losses.MeanSquaredError object at 0x000002044DE94750>", "_timestamp": 1700573962.6844656, "_runtime": 2.5576467514038086, "_step": 0, "_wandb": {"runtime": 12}}